data/json/items/gun/300.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "id": "m2010",
+    "copy-from": "rifle_base",
+    "type": "GUN",
+    "name": "M2010 ESR",
+    "name_plural": "M2010 ESR",
+    "description": "A bolt-action, box-magazine-fed, sniper rifle chambered in the powerful .300 Winchester Magnum round.  The M2010 had completely replaced the M24 rifle series by 2017.",
+    "price": 1701000, "//" : "http://www.guns.com/2012/05/17/remington-xm2010-army-sniper-system-esr-now-in-stores/.",
```

Comment:
What is the proper field ordering then? I tried to copy it from other items in this folder.